graphene_django/rest_framework/mutation.py

```diff
@@ -124,8 +126,11 @@ def __init_subclass_with_meta__(
     def get_serializer_kwargs(cls, root, info, **input):
         lookup_field = cls._meta.lookup_field
         model_class = cls._meta.model_class
-
         if model_class:
+            for input_dict_key, maybe_enum in input.items():
+                if isinstance(maybe_enum, Enum):
+                    choicefield_enum = maybe_enum
+                    input[input_dict_key] = choicefield_enum.value
```

Comment:
super minor: I think this could remove the intermediate variable to simplify, since it should be clear

```suggestion
                    input[input_dict_key] = maybe_enum.value
```